src/devTools.js

```diff
@@ -138,6 +143,7 @@ function liftReducer(reducer, initialState) {
         currentStateIndex++;
       }
       stagedActions = [...stagedActions, liftedAction.action];
+      timestamps = [...timestamps, Date.now()];
       break;
```

Comment:
No worries, this is just prevalent for me because I want to work with it in the slider monitor. :smile:

Yeah, I agree recording timestamps in ```liftAction``` is good. I just want to be able to start a replay from app startup or RESET, etc, which would mean recording timestamps in those instances. That way, the ```timestamps``` array is always is corollated with ```computedStates``` and ```stagedActions```. Hope that makes sense.